roles/bootstrap-os/tasks/main.yml

```diff
@@ -21,9 +21,13 @@
 - name: Gather nodes hostnames
   setup:
     gather_subset: '!all'
-    filter: ansible_hostname
+    filter: ansible_*
```

Comment:
Gather all happens later. Why did you change this?

Reply:
Because `ansible_os_family` is used in the hostname task to check if we are on CoreOS